src/fidesops/core/config.py

```diff
@@ -214,6 +243,14 @@ class Config:  # pylint: disable=C0115
         f'Startup configuration: pii logging = {os.getenv("FIDESOPS__LOG_PII") == "True"}'
     )
 
+    def log_all_config_values(self) -> None:
+        """Output DEBUG logs of all the config values."""
+        for settings in [self.database, self.redis, self.security, self.execution]:
+            for key, value in settings.dict().items():
+                logger.debug(
+                    f"Using config: {NotPii(settings.Config.env_prefix)}{NotPii(key)} = {NotPii(value)}"
```

Comment:
In order for vals to be masked properly, we have to use `%` string interpolation instead of `f`. This is because we intercept the Python `LogRecord` when we check for PII args, and `LogRecord` is only compatible with `%` str interpolation- https://docs.python.org/3/library/logging.html#logrecord-attributes. We could not find a solution to intercept `f` formatted strings.

So, even though we're saying `NotPii` here, mind updating to `%` for consistency in approach?

Reply:
Ah, OK I see. Will do

Reply:
OK, updated